### PR TITLE
Create content-management.md

### PR DESCRIPTION
**Pull request by teizenman**
## Status
Ready

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

### CODE REVIEW

File: content-repo/extra-docs/articles/XSOAR-Build.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+[Example file](https://raw.githubusercontent.com/demisto/content/aace565faff531f09a42268b897d629981e69b08/Packs/XSOARbuild/docs-files/xsoar_config.json)
+
+---
+
+### The `Configuration Setup` Incident type
+#### Custom Fields:
+3. `Configuration File Source` - The source of the configuration file.
+3. `Custom Packs Source` - The source of the custom packs.
+1. `Configuration File Path` - The relative path within the repository to the `xsoar_config.json` file. (Optional)
+2. `Branch Name` - The branch from which to fetch the file. Default is the main branch. (Optional)
```
Comment (yaakovi) [inline content-repo/extra-docs/articles/XSOAR-Build.md:24]: it's not clear what is the difference between `File Source` and `File Path`
```suggestion
1. `Configuration File Source` - The source of the configuration file.
2. `Custom Packs Source` - The source of the custom packs.
3. `Configuration File Path` - The relative path within the repository to the `xsoar_config.json` file. (Optional)
4. `Branch Name` - The branch from which to fetch the file. Default is the main branch. (Optional)
```

File: content-repo/extra-docs/articles/XSOAR-Build.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+#### Layout:
+![layout.png](https://raw.githubusercontent.com/demisto/content/aace565faff531f09a42268b897d629981e69b08/Packs/XSOARbuild/docs-files/layout.png)
```
Comment (yaakovi) [inline content-repo/extra-docs/articles/XSOAR-Build.md:27]: better to have an image with data

File: content-repo/extra-docs/articles/XSOAR-Build.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+---
+
+### The `Configuration Setup` Playbook
+This playbook will manage the entire configuration process and needs to run through a `Configuration Setup` incident type.<br />
+It consists of five sections:
```
Comment (yaakovi) [inline content-repo/extra-docs/articles/XSOAR-Build.md:33]: maybe stages?
```suggestion
It consists of five stages:
```

File: content-repo/extra-docs/articles/XSOAR-Build.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+1. Fetching the configuration file and loading it's content to the machine.
+2. Downloading and installing the custom packs.
+3. Installing marketplace packs.
```
Comment (yaakovi) [inline content-repo/extra-docs/articles/XSOAR-Build.md:36]: shouldn't the custom packs be installed after the marketplace packs?

File: content-repo/extra-docs/articles/XSOAR-Build.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+---
+
+### How to use the pack
+#### You can choose either to use the pack manually, or set it up to work as a scheduled job:
+1. In order to run it manually, you will need to create a new `Configurtaion Setup` type incident.
+2. In order to run it as a scheduled job, you will need to set up a new job for a `Configurtaion Setup` type incident.
+
+#### How to configure the incident/job:
+1. Choose what is the source of the configuration file for the playbook run:
+   1. Attachment - The file is already attached to the incident, and it's information located under the `File` context path.
```
Comment (yaakovi) [inline content-repo/extra-docs/articles/XSOAR-Build.md:51]:
```suggestion
   1. Attachment - The file is already attached to the incident, and its information located under the `File` context path.
```

File: content-repo/extra-docs/articles/XSOAR-Build.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+
+#### Layout:
+![layout.png](https://raw.githubusercontent.com/demisto/content/aace565faff531f09a42268b897d629981e69b08/Packs/XSOARbuild/docs-files/layout.png)
+
+---
+
+### The `Configuration Setup` Playbook
+This playbook will manage the entire configuration process and needs to run through a `Configuration Setup` incident type.<br />
+It consists of five sections:
+1. Fetching the configuration file and loading it's content to the machine.
```
Comment (yaakovi) [inline content-repo/extra-docs/articles/XSOAR-Build.md:34]:
```suggestion
1. Fetching the configuration file and loading its content to the machine.
```

**Review by yaakovi — COMMENTED** [review]
nice

**Comment by yaakovi** [comment]
@ShirleyDenkberg @kirbles19 
please review the doc as well

File: content-repo/extra-docs/articles/XSOAR-Build.md (hunk 4 above)
Comment (teizenman) [reply content-repo/extra-docs/articles/XSOAR-Build.md:36]: It's being done in parallel.

**Comment by ShirleyDenkberg** [comment]
@yaakovi @anara123 Doc review completed.  @teizenman The layout graphic needs data.

**Comment by teizenman** [comment]
> @yaakovi @anara123 Doc review completed. @teizenman The layout graphic needs data.

I changed the image, it will be updated once the pack will be merged in content.

**Review by yaakovi — APPROVED** [review]

**Pull request merged**